server/src/routes/course/crossCheck/createResult.ts

```diff
@@ -62,7 +62,7 @@ export const createResult = (_: ILogger) => async (ctx: Router.RouterContext) =>
     review: Array.isArray(inputData.review) ? inputData.review : [],
   };
 
-  if (isNaN(data.score) || data.score < 0) {
+  if (isNaN(data.score) || data.score < 0 || data.score > courseTask.maxScore) {
```

Comment:
Please create a new error message.

Reply:
If it's possible to send a score higher than maxscore from the frontend, it would be good to also have validation on the frontend side, if it's not too difficult to implement.

Reply:
I updated the PR with explicit error message and a check from front-end